### PR TITLE
update README.md with the newest additions

### PR DESCRIPTION
**Pull request by fabiangreffrath**

### CODE REVIEW

**Comment by fabiangreffrath** [comment]
in general, I'd like to convert the Features section away from formulated sentences into an actual list of bullet points, e.g. "Support for rendering with uncapped frame rate and frame interpolation has been added" --> "Rendering with uncapped frame rate and frame interpolation". What do you think?

**Comment by rfomin** [comment]
> Also, what's missing, what's too much?

What if we rewrite the "General" section of "Key Features"?

> The code has been made 64-bit and big-endian compatible.

This is too technical.

> The code has been ported to SDL2, so the game scene is now rendered to screen using hardware acceleration if available.

Some users still think we have hardware accelerated 3D rendering, I think we can remove that. 

> The build system has been ported to CMake with support for building on Linux and Windows (the latter using either vcpkg or a MSYS2 environment).

It's for developers only.

Maybe we should make the readme more similar to the Crispy Doom readme?

**Comment by fabiangreffrath** [comment]
Yes, you are right. The README is from a time when the changes to winmbf were still rather punctual. 😁

**Comment by ceski-1** [comment]
I would suggest a rewrite that organizes items by importance and trim out of date or less important information. Also, under "Releases", there is a link to a wiki version of the changelog that is very out of date.

**Comment by rfomin** [comment]
> Also, under "Releases", there is a link to a wiki version of the changelog that is very out of date.

I suggest deleting the old changelog, I don't think we want to waste time updating it.

File: README.md [hunk 1]
```diff
@@ -156,6 +159,8 @@ This is how this project was born.
 
 Many additions and improvements to this source port were taken from fraggle's [Chocolate Doom](https://www.chocolate-doom.org/wiki/index.php/Chocolate_Doom), taking advantage of its exceptional portability, accuracy and compatibility.
 
+Special thanks to @rfomin and @ceski-1 for implementing the more advanced features of this port, and @JNechaevsky and @MrALaux for helping to port back some useful features from their own source port projects!
```
[the remark this answers is not in the document]
Comment (fabiangreffrath) [reply README.md:162]: Oops. Well, no problem, this was just a first draft anyway. 😁

File: README.md [hunk 2]
```diff
@@ -34,67 +34,40 @@ If you turn the [Doom logo upside down](https://www.reddit.com/r/Doom/comments/8
...
- * The project strives for full demo compatibility for all supported complevels. Furthermore, this port offers quality-of-life features like e.g. a demo progress bar, demo playback warp and skip as well as continued demo recording.
- * The SPECHITS, REJECT, INTERCEPTS and DONUT overflow emulations have been ported over from Chocolate Doom / PrBoom+, allowing for some more obscure Vanilla demos to keep sync.
+ * Tall textures and sprites in DeePsea format.
+ * Unlimited extra states, sprites, mobjtypes and sounds for use in Dehacked patches (supporting the "DEHEXTRA" and "DSDHacked" specs).
+ * In-game music changing using MUSINFO.
+ * UMAPINFO support, compliant to Rev 2.2 of the [spec](https://github.com/kraflab/umapinfo).
+ * MBF21 compatibility level, compliant to Rev 1.4 of the [spec](https://github.com/kraflab/mbf21).
+ * SMMU-style swirling animated flats.
+ * Customization the extended Boom HUD using the [WOOFHUD](https://github.com/fabiangreffrath/woof/blob/master/docs/woofhud.md) lump.
```
Comment (MrAlaux) [inline README.md:64]: The wording is a bit off here: I assume you wanted to write "Customization **_of_** the [...]".

Comment (fabiangreffrath) [reply README.md]: Yes, right. Sometimes the exact same sentence seems to read differently in a code editor and in a browser. 🤔

**Pull request merged**